examples/multithread_cached_read.c

```diff
@@ -143,7 +144,7 @@ void thread_func(void *data)
 				
 		end = time_ms();
 		
-		fprintf(stderr,"%ld Thread %d got result %d with return code %s in %dms\n",time_ms(),tid,value,decode_error(rc),(int)(end-start));
+		fprintf(stderr,"%" PRId64 " Thread %d got result %d with return code %s in %dms\n",time_ms(),tid,value,decode_error(rc),(int)(end-start));
```

Comment:
This is the format specifier for int64_t from inttypes.h, GCC was complaining about %ld